content/pages/community/events.md

```diff
@@ -38,10 +38,11 @@ page](https://wiki.xmpp.org/web/Sprints).
 
 ### Upcoming
 
-* [Brussels, January 30th, 2019](https://wiki.xmpp.org/web/Sprints/2019_January_Brussels) (around FOSDEM)
 * [Berlin, March 30-31st, 2019](https://wiki.xmpp.org/web/Sprints/2019_March_Berlin)
+* Your event here!
```

Comment:
Can we make that text link to the github edit page for this page?

\[Your event here!\](https://github.com/xsf/xmpp.org/edit/master/content/pages/community/events.md)

Reply:
Good idea!